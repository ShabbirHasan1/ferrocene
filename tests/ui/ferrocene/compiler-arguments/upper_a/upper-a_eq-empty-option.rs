// The option of -A starts immediately, without requiring any equal symbol
//
// check-pass
// compile-flags: -A=

fn main() {}

// ferrocene-annotations: um_rustc_A
