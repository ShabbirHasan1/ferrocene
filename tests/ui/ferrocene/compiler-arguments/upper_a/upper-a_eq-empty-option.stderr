warning[E0602]: unknown lint: `=`
   |
   = note: requested on the command line with `-A =`
   = note: `#[warn(unknown_lints)]` on by default

warning[E0602]: unknown lint: `=`
   |
   = note: requested on the command line with `-A =`

warning[E0602]: unknown lint: `=`
   |
   = note: requested on the command line with `-A =`

warning: 3 warnings emitted

For more information about this error, try `rustc --explain E0602`.
